x-pack/plugins/encrypted_saved_objects/server/saved_objects/encrypted_saved_objects_client_wrapper.ts

```diff
@@ -200,7 +200,7 @@ export class EncryptedSavedObjectsClientWrapper implements SavedObjectsClientCon
   public async update<T>(
```

Comment:
Haha - so to be honest, I missed it 😬 

But - this is actually a problem.
We can't seem to change the deep `Partial<T>` to be `T` in the `bulkUpdate` signature (a problem we didn't have with `update`... I think because it isn't contained inside of an `Array` so the downstream type is a subset of the one upstream).

This means - we can't actually change `bulkUpdate` to require a `T` instead of `Partial<T>` without changing ESO to *not* implement `SavedObjectsClientContract`.

Any thoughts? @legrego as well? :/